code/modules/factory/parts.dm

```diff
@@ -316,3 +316,113 @@ GLOBAL_LIST_INIT(howitzer_shell, list(
 /obj/item/factory_part/howitzer_shell_tfoot/Initialize()
 	. = ..()
 	recipe = GLOB.howitzer_shell
+
+GLOBAL_LIST_INIT(swat_mask, list(
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_COMPRESSOR, STEP_ICON_STATE = "cutplate"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_GALVANIZER, STEP_ICON_STATE = "steelingot"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_CUTTER, STEP_ICON_STATE = "req_bag4"),
+	))
+
+/obj/item/factory_part/swat_mask
+	name = "SWAT mask"
+	desc = "An unfinished SWAT mask assembly"
+	result = /obj/item/clothing/mask/gas/swat
+
+/obj/item/factory_part/swat_mask/Initialize()
+	. = ..()
+	recipe = GLOB.swat_mask
+
+GLOBAL_LIST_INIT(optical_imager, list(
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_HEATER, STEP_ICON_STATE = "hotplate"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_COMPRESSOR, STEP_ICON_STATE = "roundplate"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_GALVANIZER, STEP_ICON_STATE = "steelingot"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_FLATTER, STEP_ICON_STATE = "unfinished_imager"),
+	))
+
+/obj/item/factory_part/optical_imager
+	name = "Optical Imagers"
+	desc = "An unfinished pair of Optical Imagers assembly"
+	result = /obj/item/clothing/glasses/night/imager_goggles
+
+/obj/item/factory_part/optical_imager/Initialize()
+	. = ..()
+	recipe = GLOB.optical_imager
+
+GLOBAL_LIST_INIT(meds, list(
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_FLATTER, STEP_ICON_STATE = "uncutplate"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_COMPRESSOR, STEP_ICON_STATE = "req_bag3"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_DRILLER, STEP_ICON_STATE = "req_bag1"),
+	list(STEP_NEXT_MACHINE = FACTORY_MACHINE_GALVANIZER, STEP_ICON_STATE = "req_bag2"),
+	))
+
+/obj/item/factory_part/med_rig
+	name = "M276 pattern medical storage rig"
+	desc = "An unfinished M276 pattern medical storage rig"
+	result = /obj/item/storage/belt/rig/medical
+
+/obj/item/factory_part/med_rig/Initialize()
+	. = ..()
+	recipe = GLOB.meds
+
+/obj/item/factory_part/med_lifesaver
+	name = "M276 pattern lifesaver bag"
+	desc = "An unfinished M276 pattern lifesaver bag"
+	result = /obj/item/storage/belt/lifesaver/full
```

Comment:
I don't know if allowing full medical belts to be made is a good thing balance wise, but I don't exactly see it being used much so it shouldn't be a major issue.

Reply:
Mass production of meds seems alright. Most medbelts aren't even used shipside, this just offers the option and is quite expensive though through mass production the cost makes up for itself.

Reply:
> Mass production of meds seems alright. Most medbelts aren't even used shipside, this just offers the option and is quite expensive though through mass production the cost makes up for itself.


This is actually one hell of a balance shake up. First thing was medication being limited yet requiring a lot of annoying steps to put em' planetside AND make it easy to carry. But providing a full belt with ease through factorio would allow marines to sustain themselves far easier.

I won't say too much against it but I still don't think it's that much of a good idea unless we're going for a faster pace of combat which... TGMC has been going down the road of.

Reply:
Changed belts to kits+splints